hicommonwealth/commonwealth
# Admin panel chain node update

## Link to Issue
Closes: #8630 

## Description of Changes
- Growth is now able to create new EVM chains in the super admin panel. 
- 
## "How We Fixed It"

-added frontend functionality so that only EVM chains are able to be entered.
-added copy and an outside link for chainlist.org
**NOTE: previous logic for dealing with other chains outside of EVM was kept**
## Test Plan
For adding a chain node:
- go to `admin-panel` 
- open another tab through the chainlist.org link
- get all the info to add a chain from there
- add info and click Submit
- confirm that the chain node you entered now shows up in the create community dropdown

For adding a chain to a community:
- in admin panel search for a community
- select a chain 
- click select and confirm in the db that the chain_node_id has been updated to the new chain id

**NOTE: The community search function searches for the id of the community, not the name, therefore it will most likely be all lowercase. However this isn't always the case, so keep this in mind when searching for a community. I have made a ticket to change this to also look for the name as well as the id.**
